src/applications/yellow-ribbon/helpers/index.js

```diff
@@ -13,3 +14,30 @@ export const normalizeResponse = response => ({
   })),
   totalResults: response?.meta?.count,
 });
+
+export const updateQueryParams = (
+  history = window.history,
+  location = window.location,
+  queryParamsLookup = {},
+) => {
+  // Derive the current query params.
+  const queryParams = new URLSearchParams(location.search);
+
+  // Set/Delete query params.
+  Object.keys(queryParamsLookup).forEach(key => {
+    // Derive the value.
+    const value = queryParamsLookup[key];
+
+    // Set the query param.
+    if (value) {
+      queryParams.set(key, value);
+      return;
+    }
+
+    // Remove the query param.
+    queryParams.delete(key);
+  });
+
+  // Update the URL with the new query params.
+  history.replaceState({}, '', `${location.pathname}?${queryParams}`);
```

Comment:
I'm wondering if the `manifest.json`'s `rootUrl` could be used here instead of `location.pathname`

Reply:
That makes sense! Though this would need updating if we were to ever add subpages (e.g. comparison functionality) later on, so perhaps this would be a more maintainable solution? What would the advantages to using the `manifest.json`'s `rootUrl` you think? 🙂

Reply:
Just that `location` is a global, so might be better if isn't tied to that, for unit tests or general portability